ocaml/xapi/xapi.ml

```diff
@@ -1422,6 +1418,10 @@ let server_init () =
                 Xapi_host.write_uefi_certificates_to_disk ~__context
                   ~host:(Helpers.get_localhost ~__context)
             )
+          ; ( "writing init complete"
```

Comment:
Add a comment that this has to stay the last entry and new entries have to be added before.